Dockerfile

```diff
@@ -129,3 +129,12 @@ CMD ["run"]
 
 # HTTP is 8080, JMX is 9001, Hazelcast is 5701-5710, Ignite is 47100, REST for Kafka is 7003
 EXPOSE 8080 9001 5701-5710 47100 7003
+
+# *****Target for test environment*****
+
+FROM release as qualitytest
+RUN mkdir /tests
+RUN chmod 777 /tests
+COPY /tests /tests
+
+FROM release
```

Comment:
Yes, this is the default build option. So, when we perform "docker build -t imagename ." this will look for the last FROM instruction to build the image in case of multi stage builds. 

We can update ReadMe to provide an understanding on that instruction.